### PR TITLE
Update wall_e_install.sh

### PR DESCRIPTION
**Pull request by SuperChamp234**
Updated wall-E Installation Script to bump up ESP-IDF version. Add user to dialout group, to allow the user to access the `/dev/tty*` ports

### CODE REVIEW

**Comment by Jamm02** [comment]
PR seems good, add an instruction to restart the machine too as adding tty and dialout for the user requires restarting the PC.

**Comment by SuperChamp234** [comment]
Added 👍🏽

File: wall_e_install.sh [hunk 1]
```diff
@@ -32,15 +33,15 @@ mkdir -p "$HOME/esp"
...
+git clone -b v4.4.3 --recursive https://github.com/espressif/esp-idf.git
 pushd esp-idf
 ./install.sh
 popd
 popd
 
 #Check if installation is successfull 
 . $HOME/esp/esp-idf/export.sh 
-idf.py --version | (grep "v4.2" && echo "Installation successfull") \
+idf.py --version | (grep "v4.2" && echo "Installation successfull! Please restart your computer for the changes to take effect.") \
```
Comment (gautam-dev-maker) [inline wall_e_install.sh:44]: * shouldn't it be `grep "4.4.3"` ?
* Why is there a need for restart ?

Comment (VedantParanjape) [reply wall_e_install.sh:44]: In the CI we are using release-v4.4, shouldn't that be used ?

Comment (Jamm02) [reply wall_e_install.sh:44]: @SuperChamp234 remove the usermod part entirely, instead explain to the attendees why giving port access is necessary and why it is needed.

Comment (gautam-dev-maker) [reply wall_e_install.sh:44]: > In the CI we are using release-v4.4, shouldn't that be used ?

`release-v4.x.x` means pre-release versions, while cloning the tags like `v4.x.x` represents the released version.
We should update our CI to use the tag `v4.4.3`

Comment (VedantParanjape) [reply wall_e_install.sh:44]: > > In the CI we are using release-v4.4, shouldn't that be used ?
> 
> `release-v4.x.x` means pre-release versions, while cloning the tags like `v4.x.x` represents the released version. As quoted in esp-idf docs :
> 
> > If a required feature is not yet available in a stable release, but you do not want to use the master branch, it is possible to check out a pre-release version or a release branch.
> 
> We should update our CI to use the tag `v4.4.3`

No, I mean this `git clone -b release/v4.2` -> release/v4.4

**Pull request merged**